pkg/cache/cache_test.go

```diff
@@ -101,7 +101,7 @@ func TestCacheClusterQueueOperations(t *testing.T) {
 							Resources: map[corev1.ResourceName]*ResourceQuota{
 								corev1.ResourceCPU: {
 									Nominal:        10_000,
-									BorrowingLimit: pointer.Int64(10_000),
+									BorrowingLimit: ptr.To(int64(10_000)),
```

Comment:
```suggestion
									BorrowingLimit: ptr.To[int64](10_000),
```

Should we use type argument instead of casting?